backends/events/sdl/sdl-events.h

```diff
@@ -59,7 +59,9 @@ class SdlEventSource : public Common::EventSource {
 
 	struct KbdMouse {
 		int16 x, y, x_vel, y_vel, x_max, y_max, x_down_count, y_down_count;
+		static const int16 multiplier = 16;
```

Comment:
I like the const int most. So I will keep it like that if you don't mind. I like having all the _km - relevant stuff together.